admin-dev/themes/default/template/controllers/cart_rules/form.js

```diff
@@ -93,7 +93,7 @@ function updateProductRuleShortDescription(item) {
   else $(`#${id2}_match`).val(length);
 }
 
-const restrictions = new Array('country', 'carrier', 'group', 'cart_rule', 'shop');
+var restrictions = new Array('country', 'carrier', 'group', 'cart_rule', 'shop');
```

Comment:
I'm afraid about what will happened if we merge it on 1.7.8.x

I think using `window` object is maybe more appropriate